### PR TITLE
Drink fixes part 2 eclectic boogaloo

### PR DESCRIPTION
**Pull request by FL-OZ**
maybe this time i wont actually break something because i tested it?

### CODE REVIEW

File: Content.Server/GameObjects/Components/Nutrition/DrinkComponent.cs [hunk 1]
```diff
@@ -147,8 +145,7 @@ private bool TryUseDrink(IEntity target)
             var split = _contents.SplitSolution(transferAmount);
             if (stomachComponent.TryTransferSolution(split))
             {
-                if (_useSound == null) return false;
```
Comment (Zumorica) [inline Content.Server/GameObjects/Components/Nutrition/DrinkComponent.cs:150]: Why did you remove this check?

Comment (FL-OZ) [reply Content.Server/GameObjects/Components/Nutrition/DrinkComponent.cs:150]: oh wait, good idea actually
